src/app/contextBroker/contextBroker.cpp

```diff
@@ -271,6 +274,7 @@ bool            statNotifQueue;
 #define PIDPATH             _i "/tmp/contextBroker.pid"
 #define IP_ALL              _i "0.0.0.0"
 #define LOCALHOST           _i "localhost"
+#define MAX_PERIOD          (3600 * 24 * 31)  // Once a month ...
```

Comment:
MAX_PERIOD -> MAX_SUMMARY_TRACE_PERIOD (or maybe it is used for some other thing?)

In addition, if there is a maximum value for `-logSummary`, it should be mentioned in the cli.md file.